android/java/org/chromium/chrome/browser/ntp/sponsored/SponsoredImageUtil.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
```

Comment:
We post the license header in that format
/* Copyright (c) 2019 The Brave Authors. All rights reserved.

 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this file,
 * You can obtain one at http://mozilla.org/MPL/2.0/. */

Reply:
Just open and close comment and the same rows as you start and end the license